### PR TITLE
MINOR: Fix upgrade instructions for 3.8 and 3.9

### PR DESCRIPTION
**Pull request by jlprat**
We forgot to add the instructions to upgrade for 3.9 and 3.8.

*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by jlprat** [comment]
Related PRS: 
https://github.com/apache/kafka/pull/17449 for the 3.8 branch
https://github.com/apache/kafka/pull/17447 for the 3.9 branch

**Review by chia7712 — APPROVED** [review]

**Comment by jlprat** [comment]
Thanks @chia7712 !

File: docs/upgrade.html [hunk 1]
```diff
@@ -141,6 +141,67 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
...
+        <li>Once the cluster's behavior and performance has been verified, bump the metadata.version by running
+            <code>
+                bin/kafka-features.sh upgrade --metadata 3.9
+            </code>
+        </li>
+        <li>Note that cluster metadata downgrade is not supported in this version since it has metadata changes.
+            Every <a href="https://github.com/apache/kafka/blob/trunk/server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java">MetadataVersion</a>
+            after 3.2.x has a boolean parameter that indicates if there are metadata changes (i.e. <code>IBP_3_3_IV3(7, "3.3", "IV3", true)</code> means this version has metadata changes).
+            Given your current and target versions, a downgrade is only possible if there are no metadata changes in the versions between.</li>
+    </ol>
```
Comment (chia7712) [inline docs/upgrade.html:204]: nit: could you please add a new line?

Comment (jlprat) [reply docs/upgrade.html:204]: Added

**Pull request merged**